### ISSUE TITLE
Not hitting break points after update

### ISSUE THREAD

**Issue by kyledenton** [Microsoft/vscode#45870]
Issue Type: <b>Bug</b>

After updating to Version 1.21.1, we cannot hit breakpoints in our components.

VS Code version: Code 1.21.1 (79b44aa704ce542d8ca4a3cc44cfca566e7720f1, 2018-03-14T14:46:47.128Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz (8 x 3392)|
|Memory (System)|15.91GB (7.92GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe C:\Repos\IVS|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (30)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
swagger-viewer|Arj|1.5.0
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
vscode-markdownlint|Dav|0.14.0
vscode-eslint|dba|1.4.7
gitlens|eam|8.1.1
EditorConfig|Edi|0.12.1
tslint|eg2|1.0.28
auto-rename-tag|for|0.0.15
beautify|Hoo|1.3.0
output-colorizer|IBM|0.1.2
ng2-dev|jew|0.0.7
docthis|joe|0.6.0
Angular2|joh|2.14.0
ng-bootstrap-snippets|ktr|1.0.0
Angular-BeastCode|Mik|5.2.21
swagger-doc-viewer|mim|1.0.4
csharp|ms-|1.14.0
team|ms-|1.122.0
debugger-for-chrome|msj|4.2.0
vscode-css-peek|pra|2.1.0
swaggitor|qns|0.1.1
vscode-icons|rob|7.21.0
yo|sam|0.9.3
vscode-gulp|tan|0.0.4
json2yaml|tux|0.2.0
JasmineSnippets|xab|1.0.1
JavaScriptSnippets|xab|1.5.0
html-css-class-completion|Zig|1.17.1

(1 theme extensions excluded)

</details>
Reproduces only with extensions

**Comment by weinand** [comment]
Are you using sourcemaps?
How does you launch config look like?

This is probably https://github.com/Microsoft/vscode/issues/45657#issuecomment-373556464, please see the comment there and try in the next Insiders build.

**Comment by amcasey** [comment]
I can repro consistently.  Feel free to reach out.  In my case, the BP is in a TS file, so there's a sourcemap. Launch configuration:
```
"type": "node",
"request": "attach",
"protocol": "inspector",
"timeout": 10000,
"stopOnEntry": false,
"name": "Attach (Inspector)",
"outFiles": [
    "${workspaceRoot}/out/**/*.js"
]
```

**Comment by dinhanhhuy** [comment]
same here, work with breakpoints in `server.js`, but don't hit on other files (they become transparent as soon as I start to attach process).

here the config
- start: `node --inspect-brk=0.0.0.0:9229 server.js`
- attach option:
```
{
            "type": "node",
            "request": "attach",
            "name": "Attach to Remote",
            "address": "**.***.***.***",
            "port": 9229,
            "localRoot": "${workspaceFolder}/server.js",
            "remoteRoot": "/root/***/***",
            "protocol": "inspector",
        },
```

**Comment by roblourens** [comment]
Could you all try downloading the Insiders' build and tell me whether it happens there? https://code.visualstudio.com/insiders

**Comment by amcasey** [comment]
@roblourens is there a way to revert if it doesn't work?  I'm pretty dependent on this functionality. 😄

**Comment by roblourens** [comment]
The main issue in 1.21 that I'm aware of is fixed in the Insiders version. It installs side by side and doesn't affect your stable setup.

If it still doesn't work there, I'd ask for more details and the log file.

**Comment by amcasey** [comment]
Still having problems in 
```
Version 1.22.0-insider
Commit b75321329388d74e34f79dcfcb789246e31836a8
Date 2018-03-22T06:16:52.928Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```

Very strange problems, actually.  I had five breakpoints that should all have been hit.  When I hit the first one, I delete it.  I didn't see any other BPs in that run.  I ran again, hit a different BP, and deleted it, but didn't see any other BPs.  By deleting BPs and re-running until each was the first to be hit, I was able to hit all of them (i.e. confirm that they should all have been hit in the initial run).
